sourcegraph/sourcegraph.proto

```diff
@@ -986,6 +988,22 @@ message ChangesetUpdateOp {
 	UserSpec Author = 8 [(gogoproto.nullable) = false];
 }
 
+message ChangesetMergeOp {
+	// Repo holds the RepoSpec where the Changeset to be merged is located.
+	RepoSpec repo = 1 [(gogoproto.nullable) = false];
+
+	// ID holds the ID of the changeset that is to be merged.
```

Comment:
I agree, however this seems to be the default style (23 matches for "holds the" and 6 matches for "contains the") so I matched it for consistency